subsys/mpsl/Kconfig

```diff
@@ -25,6 +25,13 @@ config MPSL_TIMESLOT_SESSION_COUNT
 	help
 	  Maximum number of timeslot sessions.
 
+config MPSL_SCHEDULER_INIT
+	bool "Initialize MPSL scheduler"
+	default y
+	help
+	  Initialize MPSL scheduler. Allows the use of all
+	  MPSL functionality.
```

Comment:
> Allows the use of all MPSL functionality.

So what MPSL functionality is not allowed or possible to use when the scheduler is disabled ?
Besides the use of the scheduler itself, of course.

If this text is unclear to me, then I assume it's also unclear to other users.

Reply:
Ok I changed this description. It schould be clear now what this configuration is for